src/hotspot/cpu/x86/macroAssembler_x86.cpp

```diff
@@ -5412,60 +5412,62 @@ int MacroAssembler::store_inline_type_fields_to_buf(ciInlineKlass* vk, bool from
   int call_offset = -1;
 
 #ifdef _LP64
+  // The following code is similar to allocate_instance but has some slightly differences,
+  // e.g. object size is always not zero, sometimes it's constant; storing klass ptr after
+  // allocating is not necessary if vk != NULL, etc. allocate_instance can not aware of these.
```

Comment:
`allocate_instance can not aware of these` -> `allocate_instance is not aware of these`